GenderPayGap.WebUI/Views/ManageOrganisations/OrganisationReportsTable.cshtml

```diff
@@ -0,0 +1,124 @@
+﻿@using GenderPayGap.WebUI.Helpers
+@using GenderPayGap.Database
+@using GenderPayGap.WebUI.Classes.Formatters
+@using GenderPayGap.Core
+@using GenderPayGap.Core.Classes
+@model GenderPayGap.WebUI.Models.Organisation.OrganisationReportsViewModel
+
+<table class="govuk-table">
+    <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Year</th>
+            <th scope="col" class="govuk-table__header">Reporting requirement</th>
+            <th scope="col" class="govuk-table__header">Report status</th>
+        </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+        @{
+            List<int> reportingYears = ReportingYearsHelper.GetReportingYears();
+
+            for (int i = 0; i < reportingYears.Count; i++)

```

Comment:
I feel like this file is horrible - it feels like a real code smell to me that there's _so much_ C# alongside the HTML here. I tried to refactor some of the complexity out, but I think a lot of it is pretty much essential so this uses the right one of the many complicated variable urls/bits of text in each place!